ast/misc.go

```diff
@@ -228,7 +268,20 @@ type ExecuteStmt struct {
 
 // Restore implements Node interface.
 func (n *ExecuteStmt) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	ctx.WriteKeyWord("EXECUTE ")
+	ctx.WriteName(n.Name)
+	if n.UsingVars != nil && len(n.UsingVars) > 0 {
```

Comment:
```suggestion
	if len(n.UsingVars) > 0 {
```

It is unnecessary to check `n.UsingVars != nil` since `len(nil) == 0`.